### PR TITLE
feat: new isolated test 930100-4 (930100 PL1) (Christian Folini)

### PR DESCRIPTION
**Pull request by dune73**
Status page test. See https://github.com/coreruleset/coreruleset/issues/3351

### CODE REVIEW

File: tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml [hunk 1]
```diff
@@ -51,3 +51,19 @@ tests:
             uri: "/?foo=0x2e.%000x2f0x2e.%00/WINDOWS/win.ini"
           output:
             log_contains: id "930100"
+  - test_title: 930100-4
+    desc: "Path Traversal Attack (<backslash>../), partially encoded"
```
Comment (RedXanadu) [inline tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml:55]:
```suggestion
    desc: "Status Page Test - Path Traversal Attack (<backslash>../), partially encoded"
```

Change to follow the convention we started in Varese.

**Comment by RedXanadu** [comment]
Confirmed that the test itself works as expected.

**Review by RedXanadu — CHANGES_REQUESTED** [review]
Test works fine. Can merge following changes.

File: tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml [hunk 2]
```diff
@@ -51,3 +51,19 @@ tests:
...
+          input:
+            dest_addr: "127.0.0.1"
+            method: "GET"
+            port: 80
+            headers:
+              Host: "localhost"
+              FoobarHeader: "0x5c0x2e./"
+              User-Agent: "OWASP CRS test agent"
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+            uri: "/"
```
[the remark this answers is not in the document]
Comment (dune73) [reply tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml:67]: Sure. Thanks.

**Review by RedXanadu — APPROVED** [review]
Discussed the `version: HTTP/1.1` tags. Present in the 'perfect example' documented tests, but very inconsistent across all existing tests, so we ignore it for now.

Merging now.

**Comment by RedXanadu** [comment]
Oops: race condition caused an automated 980xxx test to fail. Re-running…

**Comment by RedXanadu** [comment]
Test passed on second run.

Merging now.

**Pull request merged**